tools/ci_build/github/azure-pipelines/win-eager-ci-pipeline.yml

```diff
@@ -0,0 +1,118 @@
+parameters:
+- name: RunOnnxRuntimeTests
+  displayName: Run Tests?
+  type: boolean
+  default: true
+
+jobs:
+- job: 'build'
+  pool: 'Win-CPU-2019'
+  strategy:
+    matrix:
+      x64_release:
+        BuildConfig: 'RelWithDebInfo'
+        EnvSetupScript: setup_env.bat
+        buildArch: x64
+        additionalBuildFlags: --use_dnnl --build_java --build_nodejs
+        msbuildPlatform: x64
+        isX86: false
+  variables:
+    OrtPackageId: 'Microsoft.ML.OnnxRuntime'
+    MsbuildArguments: '-maxcpucount'
+    OnnxRuntimeBuildDirectory: '$(Build.BinariesDirectory)'
+    DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true    
+    setVcvars: true
+    ALLOW_RELEASED_ONNX_OPSET_ONLY: '0'
+  timeoutInMinutes: 120
+  workspace:
+    clean: all
+  steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '3.7'
+      addToPath: true
+      architecture: $(buildArch)
+
+  - task: NodeTool@0
+    inputs:
+      versionSpec: '12.x'
+      force32bit: $(isX86)
+
+  - task: JavaToolInstaller@0
+    #Our build machine doesn't have java x86
+    condition: and(succeeded(), eq(variables['buildArch'], 'x64'))
+    inputs:
+      versionSpec: '11'
+      jdkArchitectureOption: $(buildArch)
+      jdkSourceOption: 'PreInstalled'
+
+  - task: BatchScript@1
+    displayName: 'setup env'
+    inputs:
+      filename: '$(Build.SourcesDirectory)\tools\ci_build\github\windows\$(EnvSetupScript)'
+      modifyEnvironment: true
+      workingFolder: '$(Build.BinariesDirectory)'
+
+  - script: |
+     set ORT_DOXY_SRC=$(Build.SourcesDirectory)
+     set ORT_DOXY_OUT=$(Build.BinariesDirectory)\$(BuildConfig)\$(BuildConfig)
+     mkdir %ORT_DOXY_SRC% 
+     mkdir %ORT_DOXY_OUT%
+     "C:\Program Files\doxygen\bin\doxygen.exe" $(Build.SourcesDirectory)\tools\ci_build\github\Doxyfile_csharp.cfg
+     
+    workingDirectory: '$(Build.SourcesDirectory)'
+    displayName: 'API Documentation Check and generate'
+
+  - script: |
+     python -m pip install -r $(Build.SourcesDirectory)\tools\ci_build\github\windows\eager\requirements.txt
+     python -m pip list
+    workingDirectory: '$(Build.BinariesDirectory)'
+    displayName: 'Install python modules'
+  
+  - task: BatchScript@1
+    displayName: 'setup env 2'
+    inputs:
+      filename: '$(Build.SourcesDirectory)\tools\ci_build\github\windows\$(EnvSetupScript)'
+      modifyEnvironment: true
+      workingFolder: '$(Build.BinariesDirectory)'
+
+  - script: |
+     python -c "import torch; import sys; print(sys.path); print(torch.__file__)"
+     python -m pip list
+    workingDirectory: '$(Build.BinariesDirectory)'
+    displayName: 'Check python modules'
+
+  - task: PythonScript@0
+    displayName: 'Build Eager mode'
+    inputs:
+      scriptPath: '$(Build.SourcesDirectory)\tools\ci_build\build.py'
```

Comment:
Our windows pipelines do not use this script to build code.
If you want to be the first, that's fine. Please add "--parallel"

Reply:
Added parallel